heron/stmgr/src/cpp/manager/stmgr-server.h

```diff
@@ -75,6 +76,9 @@ class StMgrServer : public Server {
   virtual void HandleConnectionClose(Connection* connection, NetworkErrorCode status);
 
  private:
+  void DrainTupleSet(sp_int32 _task_id, proto::system::HeronTupleSet2* _message);
```

Comment:
As part of my cleanup, I'm going to change the TupleSet2 protobufs to remove the 2 part in future changes.